external/buildscripts/Build.bee.cs

```diff
@@ -1,18 +1,24 @@
 using System;
 using System.Collections.Generic;
 using Bee.Core;
-using Bee.Stevedore;
-using Bee.Stevedore.Program;
-using Unity.BuildSystem.NativeProgramSupport;
+using Bee.Core.Stevedore;
 
 namespace BuildProgram
 {
 	public class BuildProgram
 	{
-		private static readonly Dictionary<string, Tuple<string, string>> Artifacts = new Dictionary<string, Tuple<string, string>>();
+		private static readonly List<StevedoreArtifact> StevedoreArtifacts = new List<StevedoreArtifact>();
 
 		internal static void Main()
 		{
+			var manifest = "manifest.stevedore";
+
+			Backend.Current.StevedoreSettings = new StevedoreSettings
+            {
+                Manifest = { manifest },
+                //EnforceManifest = true,
```

Comment:
What's this commented out line do?

Reply:
"By default, the build code would still be able to freely request (say) any version of 7za-linux-x64, because the Linux artifact is not listed in the manifest. Enabling EnforceManifest instead makes it an error to reference any artifact not in the manifest, and is recommended, because it guarantees that looking at the manifest file(s) gives a complete picture of all Stevedore artifacts possibly needed to build."

I will remove this commented line, since it does not apply in this case.